bigtable-client-core-parent/bigtable-client-core/src/main/java/com/google/cloud/bigtable/config/CallOptionsConfig.java

```diff
@@ -135,6 +156,21 @@ public Builder setMutateRpcTimeoutMs(int mutateRpcTimeoutMs) {
       return this;
     }
 
+    /**
+     * The amount of time in milliseconds to wait before issuing a client side timeout for row
+     * mutation RPCs.
```

Comment:
Please clarify that this only affects bulk mutations. And similar to above explain the difference between mutateRpcTimeout

Reply:
done